llama-index-core/llama_index/core/agent/runner/base.py

```diff
@@ -628,6 +628,7 @@ async def achat(
         return chat_response
 
     @dispatcher.span
+    @trace_method("chat")
```

Comment:
I noticed this is the only function with a dispatcher span? 👀

Reply:
This was just for initial testing with stream_chat stuff. We need to expand coverage here, but not trying to change much yet until the backwards_compat stuff has been resolved. Hope that makes sense.

Reply:
We could just remove it until we handle streaming and agent runner completely with events and spans.

Reply:
thats fair, we can leave it for now